Example/Podfile.lock

```diff
@@ -17,11 +14,11 @@ PODS:
   - iOSSnapshotTestCase/SwiftSupport (6.2.0):
     - iOSSnapshotTestCase/Core
   - Paralayout (0.9.1)
-  - SnapshotTesting (1.7.2)
+  - SnapshotTesting (1.8.2)
```

Comment:
Looks like this version of SnapshotTesting doesn't build with Xcode 10. I don't think we're getting anything from Xcode 10 support at this point, so probably fine to update CI to Xcode 11-only.

Reply:
Bumped our CI to Xcode 11 across the board. Should be good now.